.github/actions/install_syft_grype/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Install Syft & Grype
+description: |
+  Installs Syft & Grype.
+inputs:
+  versionSyft:
+    description: "Version of the operator-sdk to install"
+    default: "0.66.2"
+    required: false
+  versionGrype:
+    description: "Version of Grype to install"
+    default: "0.55.0"
+    required: false
```

Comment:
Is it needed that these can be configured? Shouldn't we just pin them?

Reply:
Not really, we can also pin them. Just wrote it as a generic function.